app/retail/templates/email_unsubscribed.html

```diff
@@ -0,0 +1,52 @@
+{% comment %}
+  Copyright (C) 2018 Gitcoin Core
+
+  This program is free software: you can redistribute it and/or modify
+  it under the terms of the GNU Affero General Public License as published
+  by the Free Software Foundation, either version 3 of the License, or
+  (at your option) any later version.
+
+  This program is distributed in the hope that it will be useful,
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+  GNU Affero General Public License for more details.
+
+  You should have received a copy of the GNU Affero General Public License
+  along with this program. If not, see <http://www.gnu.org/licenses/>.
+{% endcomment %}
+{% load i18n static %}
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    {% include 'shared/head.html' %}
```

Comment:
hmm this logo looks messed up http://bits.owocki.com/5022c094c65c/Screen%20Shot%202019-10-14%20at%201.06.12%20PM.png

Reply:
The logo came from the figma design. I exported it from there

Reply:
here is a better logo http://bits.owocki.com/ec1d9a5520d0/gitcoin_logo_white%20copy%203.png